test/Dapr.Client.Test/InvokeApiTest.cs

```diff
@@ -217,11 +231,12 @@ public async Task InvokeMethodAsync_CanInvokeMethodWithNoReturnTypeAndData_Throw
 
             // Get Request and validate            
             httpClient.Requests.TryDequeue(out var entry).Should().BeTrue();
-            var envelope = await GrpcUtils.GetEnvelopeFromRequestMessageAsync<InvokeServiceEnvelope>(entry.Request);
+            var envelope = await GrpcUtils.GetEnvelopeFromRequestMessageAsync<InvokeServiceRequest>(entry.Request);
             envelope.Id.Should().Be("test");
-            envelope.Method.Should().Be("test");
-            var json = envelope.Data.Value.ToStringUtf8();
-            var typeFromRequest = JsonSerializer.Deserialize<InvokeRequest>(json);
+            envelope.Message.Method.Should().Be("test");
+            DataWithContentType data = envelope.Message.Data.Unpack<DataWithContentType>();
+            string s = data.Body.ToStringUtf8();
```

Comment:
Please use descriptive variable names rather than s and t.
use var, be consistent with usage of var.